frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/create-block-pool-modal.tsx

```diff
@@ -125,12 +127,10 @@ export const CreateBlockPoolModal = withHandlePromise((props: CreateBlockPoolMod
         <BlockPoolModalFooter
           state={state}
           dispatch={dispatch}
-          onPoolCreation={onPoolCreation}
-          onClick={onClick}
+          onSubmit={onClick}
           cancel={props.cancel}
           close={props.close}
-          actionLabel={t('ceph-storage-plugin~Create')}
-          closeLabel={t('ceph-storage-plugin~Finish')}
+          footerPrimaryAction={FooterPrimaryActions.CREATE}
```

Comment:
nit: since its footer component's prop
```suggestion
          primaryAction={FooterPrimaryActions.CREATE}
```